R/process_sc_all_home_care.R

```diff
@@ -0,0 +1,183 @@
+#' Process the all home care extract
+#'
+#' @description This will read and process the
+#' all home care extract, it will return the final data
+#' but also write this out as a rds.
+#'
+#' @param data The extract to process
+#' @param write_to_disk (optional) Should the data be written to disk default is
+#' `TRUE` i.e. write the data to disk.
+#'
+#' @return the final data as a [tibble][tibble::tibble-package].
+#' @family process extracts
+#' @import lubridate
```

Comment:
We definitely don't want to import the whole lubridate package. I couldn't see where you used it but we should be able to use `lubridate::<fun>` or possibly have `@importFrom lubridate <fun>` in `createslf-package.R`.